sbi/neural_nets/estimators/score_estimator.py

```diff
@@ -621,24 +663,30 @@ class VEScoreEstimator(ConditionalScoreEstimator):
 
     def __init__(
         self,
-        net: nn.Module,
+        net: VectorFieldNet,
         input_shape: torch.Size,
         condition_shape: torch.Size,
+        embedding_net: Optional[nn.Module] = None,
         weight_fn: Union[str, Callable] = "max_likelihood",
-        sigma_min: float = 1e-5,
-        sigma_max: float = 5.0,
+        sigma_min: float = 1e-4,
+        sigma_max: float = 10.0,
```

Comment:
as above, could you please add a brief into about these different defaults to the class docstring above?

Reply:
Ah, I see. Yes.